src/main/java/de/zalando/aruha/nakadi/config/RepositoriesConfig.java

```diff
@@ -26,8 +32,19 @@
     @Autowired
     private JdbcTemplate jdbcTemplate;
 
+    @Autowired
+    private ZookeeperConfig zookeeperConfig;
+
     @Bean
-    public EventTypeRepository eventTypeRepository() {
-        return new EventTypeDbRepository(jdbcTemplate, jsonConfig.jacksonObjectMapper());
+    public EventTypeRepository eventTypeRepository() throws Exception {
+        final EventTypeRepository dbRepo = new EventTypeDbRepository(jdbcTemplate, jsonConfig.jacksonObjectMapper());
+
+        final String connectString = zookeeperConfig.zooKeeperHolder().get().getZookeeperClient().getCurrentConnectionString();
+        final RetryPolicy retryPolicy = new ExponentialBackoffRetry(1000, 3);
+        final CuratorFramework client = CuratorFrameworkFactory.newClient(connectString, retryPolicy);
```

Comment:
@rcillo, `AdminUtils` is using not `Curator` but `ZkUtils` to work with Zookeeper.

Currently we are using the same Exhibitor/ZooKeeper for Kafka and for Nakadi, that means we should not create new CuratorFramework but use already existing one. It is not doing much besides periodically fetching list of Kafka brokers from ZooKeeper.